src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -76,6 +77,9 @@
 # The name of the feature flag which enables the App Store Connect symbol source.
 APP_STORE_CONNECT_FEATURE_NAME = "organizations:app-store-connect"
 
+# The feature which allows multiple sources per project.
+MULTIPLE_SOURCES_FEATURE_NAME = "organizations:app-store-connect-multiple"
```

Comment:
Having a constant for this seems a bit excessive, given that the string is used verbatim in a few other places. Since we have this already for the overall app store connect feature, that's fine with me, however.